modules/ROOT/pages/home-page/index.adoc

```diff
@@ -1,6 +1,15 @@
 = Home page
 :description: This section describes the Home Page of the Neo4j Ops Manager.
 
+Once running, Ops Manager can, by default, be accessed by navigating to the following location:
+
+----
+http(s)://<server>:8080
+----
+
+Replace `server` with the hostname of the machine running Ops Manager Server.
+If Ops Manager is running on the computer you are reading this on, try `http://localhost:8080`.
```

Comment:
```suggestion
If Ops Manager is running on the computer you are reading this on, try `https://localhost:8080`.
```

Reply:
Guaranteed cert error first time :D I think for localhost, using http makes sense, as I'm thinking this is likely during setup.